test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Verbose.Tests.ps1

```diff
@@ -26,4 +26,20 @@ Describe "Write-Verbose" -Tags "CI" {
 
 	$(Write-Verbose -Message "test" -Verbose:$true) 4>&1 | Should Be "test"
     }
+
+    It "Should not have added line breaks" {
+        $text = "0123456789"
+        while ($text.Length -lt [Console]::WindowWidth) {
+            $text += $text
+        }
+        $origVerbosePref = $VerbosePreference
+        $VerbosePreference = "continue"
+        try {
+            $out = Write-Verbose $text *>&1
```

Comment:
The same "2>&1".

Reply:
Verbose isn’t stream 2, I believe it’s 3, but redirection only worked when using all streams